rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -654,6 +665,8 @@ static void check_opts(const char *opt)
 		kvm_rootfs = strdup(opt + 11);
 	} else if (!strncmp(opt, "kvm-init=", 9)) {
 		kvm_init = strdup(opt + 9);
+	} else if (!strcmp(opt, "tls-server")) {
```

Comment:
tls-server should be implemented a main function of skeleton. I mean `args: [ "tls", "--server" ]` or `args: [ "tls", "--client" ]` is the right way to implement a so-call main function.